tests/e2e/operator.sh

```diff
@@ -101,12 +103,13 @@ install_operator() {
 
 	# Wait the operator controller to be running.
 	local controller_pod="cc-operator-controller-manager"
-	local cmd="kubectl get pods -n "$op_ns" --no-headers |"
+	local cmd="kubectl get pods -n '$op_ns' --no-headers |"
 	cmd+="egrep -q ${controller_pod}.*'\<Running\>'"
 	if ! wait_for_process 120 10 "$cmd"; then
 		echo "::error:: ${controller_pod} pod is not running"
 
-		local pod_id="$(get_pods_regex $controller_pod $op_ns)"
+		local pod_id
+		pod_id="$(get_pods_regex '$controller_pod' '$op_ns')"
```

Comment:
This one still needs to be treated (`"` vs. `'`)

Reply:
(It should be the last thing, consider it acked once it is addressed - I'm on PTO already, not sure when I get to it)

Reply:
@ldoktor changes applied